spec/presenters/curation_concerns/generic_file_presenter_spec.rb

```diff
@@ -26,4 +27,9 @@
     subject { presenter.to_partial_path }
     it { is_expected.to eq 'generic_files/generic_file' }
   end
+
+  describe "office_document?" do
+    subject { presenter.office_document? }
+    it { is_expected.to be false }
+  end
 end
```

Comment:
This test just shows its proxies to the Solr doc. The test for MimeTypes is in Hydra-Works